packages/ui-view/src/View/__tests__/View.test.js

```diff
@@ -230,78 +230,16 @@ describe('<View />', async () => {
 
     it('should warn when withFocusOutline is `true`, display is set to `inline`, and focusPosition is set to `offset`', async () => {
       const consoleError = stub(console, 'error')
-      const warning = 'Warning: [View] when display is set to `inline` the focus outline will only show if `focusPosition` is set to `inset`.'
+      const warning =
+        'Warning: [View] when display is set to `inline` the focus outline will only show if `focusPosition` is set to `inset`.'
       await mount(
-        <View
-          withFocusOutline
-          display="inline"
-          focusPosition="offset"
-        >
+        <View withFocusOutline display="inline" focusPosition="offset">
           <h1>Hello!</h1>
         </View>
       )
 
       expect(consoleError).to.be.calledWith(warning)
     })
-
-    it('should apply the correct focus ring depending on the border radius', async () => {
```

Comment:
Previously it was testing if the appropriate border radius class was applied to the component and since from now on we only have 1 class applied to an element (root), I wasn't sure how to test this behaviour without testing the applied CSS itself which i'm not really keen on doing. (I believe we should use VRT for these kind of tests)

Reply:
TRue, VRT should solve this issue :)